incubator/kubewatch/templates/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kubewatch.fullname" . }}-config
+  namespace: {{ .Release.Namespace }}
```

Comment:
Why are you settings the namespace explicitly? Shouldn't be necessary.

Reply:
oops.